doc/functions.sgm

```diff
@@ -149,6 +149,41 @@
             </example>
           </sect2>
 
+          <sect2 id="func.spoint_dwithin">
+            <title>
+              Point-within-distance function
+            </title>
+            <para>
+              The function
+            </para>
+            <funcsynopsis>
+              <funcprototype>
+                <funcdef><function>spoint_dwithin</function></funcdef>
+                  <paramdef>spoint <parameter>p1</parameter></paramdef>
+                  <paramdef>spoint <parameter>p2</parameter></paramdef>
+                  <paramdef>float8 <parameter>radius</parameter></paramdef>
+              </funcprototype>
+            </funcsynopsis>
+            <para>
+              returns a boolean value that signifies whether the points
+              <parameter>p1</parameter> and <parameter>p2</parameter>
+              lie within distance <parameter>radius</parameter> of each other, i.e.
```

Comment:
The units of the radius parameter should be explicitly documented. This applies to any other functions that take a radius or distance parameter as well.